web-api/src/app.js

```diff
@@ -1011,7 +1011,10 @@ app.get(
   );
   app.get('/trial-sessions', lambdaWrapper(getTrialSessionsLambda));
   app.post('/trial-sessions', lambdaWrapper(createTrialSessionLambda));
-  app.put('/trial-sessions', lambdaWrapper(updateTrialSessionLambda));
+  app.put(
+    '/async/trial-sessions',
+    lambdaWrapper(updateTrialSessionLambda, { isAsync: true }),
```

Comment:
Now that you added this `isAsync` option, shouldn't we update all of the other async endpoints in this file to have this option too?

Reply:
yes, but it will probably break a LOT of tests since right now every test expects every endpoint to just wait until done processing.  I personally would rather make a devex item to do this work.